Avoid:
    fatal: not a git repository (or any of the parent directories): .git

--- tools/install.py.orig	2021-03-06 00:46:36.000000000 -0700
+++ tools/install.py	2021-03-24 17:27:28.000000000 -0700
@@ -38,9 +38,10 @@
 
 def add_revision_file(target):
   # text=True would be better than encoding here, but it's only supported in 3.7+
-  git_hash = subprocess.check_output(['git', 'rev-parse', 'HEAD'], encoding='utf-8').strip()
-  with open(os.path.join(target, 'emscripten-revision.txt'), 'w') as f:
-    f.write(git_hash + '\n')
+  #git_hash = subprocess.check_output(['git', 'rev-parse', 'HEAD'], encoding='utf-8').strip()
+  #with open(os.path.join(target, 'emscripten-revision.txt'), 'w') as f:
+  #  f.write(git_hash + '\n')
+  pass
 
 
 def copy_emscripten(target):
